packages/transcript-view/dev_tools/transcript-view-dev-options.ts

```diff
@@ -0,0 +1,316 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  TemplateResult,
+  CSSResult,
+} from 'lit-element';
+
+import './search-results-switcher';
+
+// these are dev tools so I don't think we need coverage for them
+/* istanbul ignore */
+
+@customElement('transcript-view-dev-options')
+export default class TranscriptViewDevOptions extends LitElement {
+  @property({ type: Number }) transcriptHeight = 200;
+
+  @property({ type: Number }) topContextHeight = 50;
+
+  @property({ type: Number }) bottomContextHeight = 50;
+
+  @property({ type: Number }) currentTime = 0;
+
+  @property({ type: Boolean }) topContextVisible = true;
+
+  @property({ type: Boolean }) bottomContextVisible = true;
+
+  @property({ type: Boolean }) autoScroll = true;
+
+  @property({ type: Boolean }) showContextZones = true;
+
+  @property({ type: Boolean }) timerRunning = false;
+
+  private timer = -1;
+
+  private timerDelay = 1000;
+
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <div>
```

Comment:
Question: are the `<div>`s for display purposes? if not, do we need?

Reply:
I could get rid of these. This file is just a dev tool (not used for the actual `transcript-vew`, just for testing its functionality) so I didn't spend much time on it 😅